packages/manager/modules/pci/src/projects/project/storages/objects/onboarding/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
There are some empty translations. Is it needed?

Reply:
Yes, to avoid to display translation key instead of nothing. We haven't all content for guides.